docs/examples/index.md

```diff
@@ -10,3 +10,4 @@ Here are the current examples:
 - [Knapsack](knapsack.md)
 - [Matrix Traversal](matrix_traversal.md)
 - [Excavation](excavation.md)
```

Comment:
Can you remove Matrix Traversal and Excavation from this list? I forgot to do it when I removed it from the navigation bar.

Reply:
Done.